css/custom-styles/_posts.scss

```diff
@@ -0,0 +1,80 @@
+.post-preview {
+  @include u-border-top("1px", "gray-30");
+  @include u-margin-y(3);
+  @include u-measure(6);
+
+  &__title,
+  &__content,
+  &__header,
+  .post-category {
+    @include u-margin-y(2);
+  }
+
+  &__header {
+    @include u-margin-top(3);
+  }
+
+  &__title {
+    @include typeset("lang", "lg", 3);
+    @include u-text("bold");
+
+    a {
+      color: inherit;
+
+      &:hover {
+        color: color($site-primary);
+      }
+    }
+  }
+
+  &__content p {
+    @include u-font("lang", "xs");
+  }
```

Comment:
**polish**: This doesn't make much of a difference and could mean one less style to maintain.

Reply:
Fair. I like that it adds a tiny bit more visual hierarchy, but happy to remove it if that is the team's preference.  It is not a must-have for me.

Reply:
It's really a tiny difference of 15.04px → 16px.